amgut/db/patches/0033.sql

```diff
@@ -0,0 +1,72 @@
+-- Feb 4, 2016
+-- Create consent revoked table
+CREATE TABLE ag.consent_revoked ( 
+    ag_login_id          uuid  NOT NULL,
+    participant_name     varchar  NOT NULL,
+    participant_email    varchar  NOT NULL,
+    date_revoked         date DEFAULT current_date NOT NULL,
+    CONSTRAINT idx_consent_revoked UNIQUE ( ag_login_id, participant_name ) 
+ );
+
+CREATE INDEX idx_consent_revoked_0 ON ag.consent_revoked ( ag_login_id );
+
+ALTER TABLE ag.consent_revoked ADD CONSTRAINT fk_consent_revoked FOREIGN KEY ( ag_login_id ) REFERENCES ag.ag_login( ag_login_id );
+
+-- Add IBD diagnosis type as retired question
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british, retired) VALUES
+(161, 'IBD_DIAGNOSIS', 'Which type of IBD?', 'Which type of IBD?', TRUE);
+
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (146, 'SINGLE');
+
+INSERT INTO survey_response (american, british) VALUES ('Ulcerative colitis', 'Ulcerative colitis');
+INSERT INTO survey_response (american, british) VALUES ('Crohn''s disease', 'Crohn''s disease');
```

Comment:
I don't think this will render properly?  Don't you need backslashes here for the quotes?